packages/components/src/Icons/MenuBookRounded.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import createSvgIcon from "../SvgIcon/utils/createSvgIcon";
+
+export default createSvgIcon(
+  <React.Fragment>
```

Comment:
Can this be `<>`? `SentimentDissatisfiedRounded.tsx` and `SentimentSatisfiedRounded.tsx` also have `React.Frament`s.

Reply:
Oh and `CategoryRounded.tsx` and `RecordVoiceOverRounded.tsx`.

Reply:
ah good catch, these were just copied over